src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_RootScreen.tsx

```diff
@@ -299,14 +301,24 @@ export const Profile_RootScreen = ({navigation}: ProfileProps) => {
             onPress={() => navigation.navigate('Profile_LanguageScreen')}
             testID="languageButton"
           />
-          <LinkSectionItem
-            text={t(
-              ProfileTexts.sections.settings.linkSectionItems.privacy.label,
-            )}
-            label="new"
-            onPress={() => navigation.navigate('Profile_PrivacyScreen')}
-            testID="privacyButton"
-          />
+          {isBeaconsEnabled ? (
```

Comment:
Haven't tested, but is it possible to just use `label={isBeaconsEnabled ? "new" : undefined}` instead of duplicating it?

Reply:
Absolutely, good idea 🙌